docs/fundamentals/flow/index.md

```diff
@@ -1,41 +1,50 @@
 (flow-cookbook)=
 # Flow
 
-{class}`~jina.Flow` ties Executors together into a processing pipeline to perform a bigger task, like indexing or querying a 
-dataset. Documents "flow" through the created pipeline and are processed by Executors.
-`Flow` also provides synchronization mechanisms to manage dependencies between executors and their order.
+A {class}`~jina.Flow` orchestrates Executors into a processing pipeline to build a neural search application.
+Documents "flow" through the created pipeline and are processed by Executors.
 
 
+The most important methods of the `Flow` object are the following:
 
+| Method                             | Description                                                                                                                                  |
+|------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| `.add()`                           | Add an Executor to the `Flow`                                                                                                                |
+| `.start()`                         | Starts the `Flow`. This will start all its Executors and check if they are ready to be used.                                                 |
+| `.stop()`                          | Stops the `Flow`. This will stop all its Executors.                                                                                          |
```

Comment:
```suggestion
| `.stop()`                          | Stops the Flow. This will stop all its Executors.                                                                                          |
```